tutorials/intellijDebugger.md

```diff
@@ -6,7 +6,89 @@ pageNav: 3
 
 # {{ title }}
 
+Before diving into tracing code with IntelliJ IDEA’s debugger, it’s essential to understand key debugger features and their purpose.
+
 This video (from LaunchCode) gives a pretty good explanation of how to use the IntelliJ IDEA debugger.
 
 @[youtube](1bCgzjatcr4)
 
+## Starting A Debugging Session
+
+To utilise the debugger, we need to make sure that the code is running in debugging mode.
+
+### Run Debugger Mode
+
+<box type="info" seamless>
+
+**Video reference:** [3.53](https://youtu.be/1bCgzjatcr4?si=fIEvUVhXHwKQeOlc&t=233): Run your program in debugging mode to use debugging features.
```

Comment:
Is it possible to indicate a stop point too? Otherwise the reader wont know when the relevant part of the video ends.